.github/workflows/integration_test.yaml

```diff
@@ -265,11 +265,19 @@ jobs:
           name: ${{ env.CHARM_NAME }}-images
           path: ${{ inputs.working-directory }}/${{ env.CHARM_NAME }}-images
           if-no-files-found: error
+  build-multiple-charms:
+    name: Build and push multiple charms out of subdirectories
+    needs: [get-runner-image, build-charm]
+    if: ${{ needs.build-charm.outputs.charm-file == '' && !cancelled() }}
```

Comment:
Isn't the file always available?

Reply:
Not always.  Envision a repo like this:

```
my-multi-charms
└── charms
    ├── charm-a
    │   └── metadata.yaml
    └── charm-b
        └── metadata.yaml
```

if there's not a `metadata.yaml` in the working-directory, then there's no charm-file built

Reply:
Understood